internal/inventory/ASSETS.md

```diff
@@ -118,6 +119,21 @@ Identity: 9% (1/11)
 Infrastructure: 7% (2/28)
 Messaging Service: 100% (2/2)
 Private Endpoint: 100% (1/1)
+=======
+**Progress: 44% (24/54)**
+Access Management: 100% (3/3)
+Container Registry: 100% (1/1)
+Database: 100% (4/4)
+File System Service: 100% (2/2)
+Group: 100% (1/1)
+Host: 100% (1/1)
+Identity: 0% (0/9)
+Infrastructure: 8% (2/23)
+Messaging Service: 100% (2/2)
+Private Endpoint: 100% (1/1)
+Service Account: 100% (1/1)
+Service Usage Technology: 100% (2/2)
+>>>>>>> da397ba4 ([Asset Inventory][Azure] Add missing resources and ECS fields required for GA (#2954))
```

Comment:
Beware conflicts. `8.x` has some parity issues with `main` when it comes to Asset Inventory. We'll have to go in and check what's different/missing.